### PR TITLE
Datamart condition updates.

### PR DESCRIPTION
**Pull request by taylorrassmannlibertyits**

### CODE REVIEW

**Comment by jblefkove-va** [comment]
@haidong @jtaylor-va @nmoisan

**Comment by jtaylor-va** [comment]
LGTM!

**Review by haidong — DISMISSED** [review]
LGTM!

File: data-query/src/main/java/gov/va/api/health/dataquery/service/controller/condition/DatamartCondition.java [hunk 1]
```diff
@@ -122,6 +134,7 @@ private void setEtlDate(String unused) {
   /** status. */
   public enum ClinicalStatus {
     active,
+    inactive,
```
Comment (jblefkove-va) [inline data-query/src/main/java/gov/va/api/health/dataquery/service/controller/condition/DatamartCondition.java:137]: Sorry, one more wrinkle... `Dstu2ConditionTransformer.clinicalStatusCode()` and `R4ConditionTransformer.clinicalStatus()` need to be updated in this PR to handle this new value

**Review by jblefkove-va — APPROVED** [review]
:shipit:

**Pull request merged**